src/pages/splash/splash.less

```diff
@@ -248,6 +248,18 @@
   }
 
   /**
+   * Maker Party Promo
+   */
+  #mp-promo {
+    .promo {
+      .promoMixin(@vikingYellow);
```

Comment:
Yes. Promos will change periodically. They will use the same core styles and select from a small set of background color options. Some day, they might appear on pages other than the splash page.